Controls/Mail/Mailman.c

```diff
@@ -13,6 +15,23 @@
  */
 void depositLetter(void) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+
+	OSSemPost(&MailboxFlag_Sem4, 
+			  OS_OPT_POST_1, 
+			  &err);
+
+	// Populate the mailbox
+	for (int i = 0; i < 256; i++) {
+		mailbox[i] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"[random() % 26];
+	}
+	mailbox[256] = '\0';
```

Comment:
The mailbox variable is declared as `char mailbox[256]`. Would this assignment be an issue?

Reply:
Which assignment? The one in the for loop or the null terminator assignment outside the loop?

Reply:
The null terminator